frontend/model/state.js

```diff
@@ -468,7 +474,12 @@ const getters = {
       .forEach(contractID => {
         const attributes = state[contractID].attributes
         if (attributes) { // NOTE: this is for fixing the error while syncing the identity contracts
-          profiles[attributes.username] = { ...attributes, contractID }
+          const username = checkedUsername(state, attributes.username, contractID)
```

Comment:
This is kinda odd, that if the check doesn't pass we're still using it, it's just `undefined` now... Is that right?

Reply:
At that point, the username can't be trusted. However, the Vuex state needs to be synchronous.